Twittimation/RandomLikeAutomaton.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Twittimation.Commands;
+using Twittimation.Http;
+using Twittimation.IO;
+using Twittimation.Twitter;
+
+namespace Twittimation
+{
+    public class RandomLikeAutomaton : IAutomaton
+    {
+        private readonly IStored<RandomLikeAutomatonData> _data;
+        private readonly IStored<Tasks> _tasks;
+        private readonly Like _like;
+        private readonly IStored<Credentials> _credentials;
+        private int _updatesToSkip = 0;
+        private const int SecondsPerHour = 60 * 60;
+
+        public RandomLikeAutomaton(IStored<RandomLikeAutomatonData> data, IStored<Tasks> tasks, Like like, IStored<Credentials> credentials)
+        {
+            _data = data;
+            _tasks = tasks;
+            _like = like;
+            _credentials = credentials;
+        }
+
+        public void Update()
+        {
+            if(_updatesToSkip == 0)
+            {
+                _updatesToSkip = 11;
+                var credentials = _credentials.Get();
+                if (credentials.AreValid)
+                {
+                    var now = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
+                    var random = new Random();
```

Comment:
Only one instance of random should be used. It should be correctly seeded.

Reply:
I am already not creating redundant random, and what you mean by correctly seeded>

Reply:
A new instance is created during every `Update` execution. You should never have more than one instance of Random per object.

For correctly seeded, you should do something like this: `var rand = new Random(Guid.NewGuid().GetHashCode());` Otherwise, there are a whole slew of issues with it not behaving randomly: https://stackoverflow.com/questions/1785744/how-do-i-seed-a-random-class-to-avoid-getting-duplicate-random-values